src/main/java/org/wildfly/security/auth/realm/ldap/LdapSecurityRealm.java

```diff
@@ -130,6 +135,7 @@
         this.credentialLoaders = credentialLoaders;
         this.credentialPersisters = credentialPersisters;
         this.evidenceVerifiers = evidenceVerifiers;
+        this.name = name;
```

Comment:
Ok, as reminded @darranl : 
> We could have an ELY issue as well then but I think it was a deliberate decision that Elytron components don't need to be named.
> Maybe a higher level discussion in the future about how we bridge that gap.

So adding the name need to be removed from PR. Sorry.